neo/Settings.py

```diff
@@ -147,9 +147,23 @@ def setup_testnet(self):
         """ Load settings from the testnet JSON config file """
         self.setup(FILENAME_SETTINGS_TESTNET)
 
-    def setup_privnet(self):
-        """ Load settings from the privnet JSON config file """
+    def setup_privnet(self, host=None):
+        """
+        Load settings from the privnet JSON config file
+
+        Args:
+            host (string): can be either an IP which will be used for seeds with standard ports and RPC
+                           or in the format of `IP:port`, which will be used as the only seed
```

Comment:
While I don't think you'll need more than 1 host on a privatenet (and thus this solution would suffice), one thing I'd like to mention just for consideration is the option to specify a `list` of 'host:port' pairs and replace the whole SEED_LIST or RPC_LIST with that. 

I am struggling a bit with the thought that this PR sort of solves the same problems as the `-c` option does, no?

Reply:
That is a good point. Perhaps for the hackathon "shared" private net use case, they should just be distributing a json file for everyone to use with `-c`?

Reply:
It just makes it easier. For instance a hackathon, you spin up a VM somewhere with privnet + neoscan + faucet. Then you just tell people to run `./prompt.py -p hackathon.cityofzion.io` and that's it. No need to copy around protocol-config json files beforehand.

True, a single host should be enough.

The question is how we'd define the RPC port, but at the same time keep things as simple as possible.

Reply:
At hackathons, every thing you don't need to do for the system setup is worth it's weight in gold. Just make it as easy as possible for participants, and I think this is it.

Reply:
Cool, you definitely know better than me on this one 😉 I'm good with keeping it simple, but I think we probably only need to support the single host scenario. For anything more complex, I would think you should just be using a config file.

Reply:
This is a great middle ground.